### PR TITLE
Disabled managed cluster test

### PR DESCRIPTION
**Pull request by dislbenn**
Disabling the managed cluster tests until the log action is fixed within the console UI/API. 

Related issue: https://github.com/stolostron/backlog/issues/19850, https://github.com/stolostron/backlog/issues/19862
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

File: .github/workflows/package-dependencies-update.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Update package dependencies versions
+
+# Set jobs to be configured and executed by schedule.
+on:
+  push:
+    branches:
+      - 'main'
+  schedule:
+    - cron: '0 8 * * Tue' # Actions will be executed at 8am for every Tuesday.
```
Comment (jlpadilla) [inline .github/workflows/package-dependencies-update.yml:9]: Can we run every weekday? Goal is to be faster than the secure engineering checks. 😄 
Maybe run earlier when there's less activity.

File: .github/workflows/package-dependencies-update.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+name: Update package dependencies versions
+
+# Set jobs to be configured and executed by schedule.
+on:
+  push:
+    branches:
+      - 'main'
```
Comment (jlpadilla) [inline .github/workflows/package-dependencies-update.yml:7]: Can we run on release branches too?

File: .github/workflows/package-dependencies-update.yml (hunk 1 above)
Comment (dislbenn) [reply .github/workflows/package-dependencies-update.yml:9]: We can run every weekday, let me adjust that with the code. 👍🏽

File: .github/workflows/package-dependencies-update.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          ref: automated-branch
```
Comment (jlpadilla) [inline .github/workflows/package-dependencies-update.yml:22]: Maybe you should use a unique branch every time this runs?

File: .github/workflows/package-dependencies-update.yml (hunk 2 above)
Comment (dislbenn) [reply .github/workflows/package-dependencies-update.yml:7]: Yes, let me make those changes to the branches.

File: tests/cypress/tests/02-search/02-pod-logs.spec.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ const clusterModes = [
   {
     label: 'Managed',
     valueFn: () => cliHelper.getTargetManagedCluster(),
-    skip: Cypress.env('SKIP_MANAGED_CLUSTER_TEST'),
+    skip: true, // Disabling the managed cluster tests until the console squad patches the issue for the logs action.
```
Comment (jlpadilla) [inline tests/cypress/tests/02-search/02-pod-logs.spec.js:26]: Can you output a message to the test output to get a reminder that this is disabled?

File: .github/workflows/package-dependencies-update.yml (hunk 2 above)
Comment (jlpadilla) [reply .github/workflows/package-dependencies-update.yml:7]: Actually, we are still testing it, so let's start with just `main` for now.

File: .github/workflows/package-dependencies-update.yml (hunk 1 above)
Comment (jlpadilla) [reply .github/workflows/package-dependencies-update.yml:9]: Also good for testing!

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**